mlflow/sagemaker/__init__.py

```diff
@@ -625,7 +626,7 @@ def _get_sagemaker_resource_unique_id():
     uuid_bytes = uuid.uuid4().bytes
     # Use base64 encoding to shorten the UUID length. Note that the replacement of the
     # unsupported '+' symbol maintains uniqueness because the UUID byte string is of a fixed,
-    # 32-byte length
+    # 16-byte length
     uuid_b64 = base64.b64encode(uuid_bytes)
```

Comment:
Maybe we could share this code in a utils class and test it?

Reply:
Done! (Added the method to `mlflow.utils` and added an `mlflow.tests.test_utils` module containing tests for this method)